Code/UnitTests/FoundationTest/Math/MathTest.cpp

```diff
@@ -381,8 +381,10 @@ EZ_CREATE_SIMPLE_TEST(Math, General)
     EZ_TEST_BOOL(ezMath::FloatToInt(12.34f) == 12);
     EZ_TEST_BOOL(ezMath::FloatToInt(-12.34f) == -12);
 
+#if EZ_ENABLED(EZ_PLATFORM_ARCH_X86) && (!defined(_MSC_VER) || _MSC_VER <= 1916) || EZ_DISABLED(EZ_PLATFORM_ARCH_X86)
```

Comment:
an undefined macro will always be evaluated as 0, so you do not need the "defined(_MSC_VER)" check